www/docs/intro.md

```diff
@@ -570,6 +571,12 @@ module.exports = {
       // venv: "path/to/venv" <- for env with installed starknet-devnet (created with e.g. `python -m venv path/to/venv`)
       venv: "<VENV_PATH>",
 
+      // use python or rust vm implementation
+      // vmLang: "python" <- use python vm
```

Comment:
it should be noted that this is the default.

Reply:
updated

Reply:
1. related to #321 
2. yes for active or custom path